src/Listeners/AddMenuPermission.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace TCG\Voyager\Listeners;
+
+use TCG\Voyager\Events\BreadAdded;
+use TCG\Voyager\Facades\Voyager;
+use TCG\Voyager\Models\Menu;
+use TCG\Voyager\Models\MenuItem;
+use TCG\Voyager\Models\Permission;
+use TCG\Voyager\Models\Role;
+
+class AddMenuPermission
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Will create Menu item and Permission after creating a BREAD item.
+     *
+     * @param BreadAdded $event
+     *
+     * @return void
+     */
+    public function handle(BreadAdded $bread)
+    {
+        $this->addPermission($bread);
```

Comment:
Can we change this so it's 2 distinct listeners? One adds the menu, and one adds the permission?  It simplifies the code a bit by making each class just a bit more atomic.

Reply:
Absolutely, great idea!

Reply:
I've committed the changes